bluebinder.c

```diff
@@ -129,8 +167,8 @@ host_write_packet(
     // before the first one was accepted.
     // but it must be lower than the binder reply callback since otherwise we might send more packets
     // than the remote can handle.
-    // it is defined in libgbinder as G_PRIORITY_DEFAULT_IDLE and must stay in sync with this
-    g_idle_add_full(G_PRIORITY_DEFAULT_IDLE + 1, waiting_for_binder_reply, proxy, NULL);
+    // it is defined in libgbinder as G_PRIORITY_DEFAULT and must stay in sync with this
+    g_idle_add_full(G_PRIORITY_DEFAULT + 1, waiting_for_binder_reply, proxy, NULL);
```

Comment:
For whatever reason, in glib lower priority numbers mean higher priority from the scheduling PoV and vice versa, which I find counter-intuitive. I suggest to use `G_PRIORITY_HIGH` or `G_PRIORITY_LOW` constants, depending on what you really want.